assets/css/general.css

```diff
@@ -26,6 +26,83 @@ h2 {
     z-index: 10000;
 }
 
+/* sass-lint:disable class-name-format */
+.position-relative {
+    position: relative;
+}
+
+.d-none {
+    display: none;
+}
+
+.d-inline-block {
+    display: inline-block;
+}
+
+.w-0 {
+    width: 0%;
+}
+
+.w-100 {
+    width: 100%;
+}
+
+.border-0 {
+    border: 0;
+}
+
+.text-center {
+    text-align: center;
+}
+
+.position-absolute {
+    position: absolute;
+}
+
+.font-weight-bold {
+    font-weight: bold;
+}
+/* sass-lint:enable class-name-format */
+
+#apertiumLogo {
+    margin-bottom: -.25em;
+}
+
+#gsocRef,
+#codeInRef {
+    text-decoration: none;
+}
+
+#gsocLogo {
+    height: 2.5em;
+}
+
+#codeInLogo {
+    height: 2.5em;
+    padding-left: .5em;
+}
+
+#bytemarkLogo {
+    margin-top: .5em;
+}
+
+#sourceForgeLogo {
+    margin-top: .25em;
+}
+
+#creativeCommonsLicence,
+#gnuGplLicence {
+    margin-bottom: 0;
+    margin-left: .5em;
+    margin-right: .5em;
+    margin-top: 0;
+}
+
+#centerBlockLogo {
```

Comment:
I think this ID could still use a bit of improvement like the other ones that you've changed since earlier.

Reply:
Hi @sushain97 how about `infoModalApertiumLogo`?